string name
string type
int8 behavior_phase
int8[5] occupancy
string target
string speech
string rob_emotion